assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -305,6 +306,12 @@ def save_as_mlflow(self):
         config_load_args = self._hf_conf.get(HF_CONF.HF_CONFIG_ARGS.value, {})
         config = self._hf_config_cls.from_pretrained(self._model_dir, local_files_only=True, **config_load_args)
         hf_conf[HF_CONF.TRAIN_LABEL_LIST.value] = list(config.id2label.values())
+        extra_pip_requirements = ["torchvision"]
```

Comment:
Do we have torchvision in converter environment???

Reply:
Yes, we do. 

Although the base image is still outdated and uses torch1.13.1 and torchvision 0.14.0
https://github.com/Azure/azureml-assets/blob/main/assets/training/model_management/environments/model-management/context/Dockerfile